### ISSUE TITLE
Rule Proposal: Warn if then() has a non-function argument

### ISSUE THREAD

**Issue by alecxe** [alecxe/eslint-plugin-protractor#62]
Example of a violation:

```js
browser.getCapabilities().then(caps) {
    // ...
});
```

Correct version:

```js
browser.getCapabilities().then(function (caps) {
    // ...
});
```

Inspired by http://stackoverflow.com/questions/41158390/any-one-help-me-to-find-out-the-unexpected-token-issue.

**Comment by alecxe** [comment]
Too ambiguous to be practical
